docs/src/literate/tutorial_kalthoff-winkler_dynfrac.jl

```diff
@@ -0,0 +1,91 @@
+# # [Kalthoff-Winkler experiment](@id tutorial_kalthoff-winkler)
+#
+# This tutorial demonstrates how to set up and run the Kalthoff-Winkler experiment using the Peridynamics.jl package.
+# The Kalthoff-Winkler experiment is a classic dynamic fracture experiment involving a pre-notched sample subjected 
+# to impact loading.
+
+# ## Introduction
+#
+# The Kalthoff-Winkler experiment is widely used to study fracture mechanics under high strain rates.
+# This setup provides valuable insights into the behavior of materials under dynamic loading conditions, making it an
+# interesting experiment in the field of fracture mechanics.
+#
+# In this tutorial, we will simulate the Kalthoff-Winkler experiment using peridynamics without the impactor. To do so,
+# first we would import the packages.
+
+# ## Import Packages
+using Peridynamics
+using GLMakie # For visualization
+using LaTeXStrings # For LaTeX string formatting
+
+# ## Geometrical Parameters
+#
+# Define the sample length `l`, width `w`, and thickness `t`, with point spacing `Δx`, horizon size `$ δ $` and crack 
+# length `a`.
+l  = 200.0E-3  # Length of the sample (meters)
+w  = 100.0E-3  # Width of the sample (meters)
+t  =   9.0E-3  # Thickness of the sample (meters)
+Δx =   0.1E-3  # Discretization size (meters)
```

Comment:
With `Δx = 0.1E-3` there will be 1.8 million points, which is too much for a tutorial. Decreasing to `Δx = 1.0E-3` should also decrease the runtime to something people trying out the tutorial can handle on their personal computers.

Reply:
Resolved on [fb99d4c]